gui/resources.xml

```diff
@@ -1173,7 +1211,7 @@ Vocabulary
       </child>
       <child>
         <object class="GtkAction" id="ShowPopupMenu">
-          <property name="label" translatable="yes" context="Menu→View (labels), Accel Editor (labels)">Popup Menu</property>
+          <property name="label" translatable="yes" context="Menu→View (labels), Accel Editor (labels)">Popup Main Menu</property>
```

Comment:
:+1:, we now have secondary menus too, but this key always triggers the main one. So it needs to say so :)